1.9/administering-clusters/deploying-a-local-dcos-universe.md

```diff
@@ -221,7 +221,7 @@ You can install and run DC/OS services on a datacenter without internet access w
 
     By default, only the `selected` packages are bundled. If you'd like to get something else, use the build your own [instructions][4].
 
-# <a name="build"></a>Installing a selected set of Universe packages
+# <a name="build"></a>Installing your own set of Universe packages
```

Comment:
Good idea, the previous wording was confusing because of the `selected` package field. Let's update the 1.8 and 1.10 docs also?